modules/openapi-generator/src/main/resources/python/python-experimental/model_utils.mustache

```diff
@@ -1041,6 +1043,16 @@ def get_oneof_instance(self, model_args, constant_args):
     # Iterate over each oneOf schema and determine if the input data
     # matches the oneOf schemas.
     for oneof_class in self._composed_schemas['oneOf']:
+        # The composed oneOf schema allows the 'null' type and the input data
+        # is the null value. This is a OAS >= 3.1 feature.
+        if oneof_class is none_type:
```

Comment:
We will only reach this code if we are deserializing a dict payload.
This is because the `__new__` method is run before the `__init__` method. And in `__new__` we return None rather than an instance of the composed class when we pass in None and oneOf contains None type.
So in that case, get_oneof_instance is never called. So in this loop, all we need to do is skip over the none_types.
Can you change this if block to be only this?
```
if oneof_class is none_type:
  # skip none_types because we are deserializing dict data. none_type deserialization is handled in the __new__ method
  continue
```

Reply:
Actually, I just added this code because otherwise the existing unit tests fail. This code IS executed when the unit tests are executed.

Reply:
Thank you for the change. I just tested it. It works.